.circleci/config.yml

```diff
@@ -10,7 +11,8 @@ jobs:
       - checkout
       - restore_cache:
           key: dropwizard-{{ checksum "pom.xml" }}
-      - run: ./mvnw --no-transfer-progress -V -B -ff -Dsurefire.rerunFailingTestsCount=1 install
+      - run: ./mvnw --no-transfer-progress -V -B -ff \
```

Comment:
it looks like that this change is causing a failure in workflow by adding extra `\`? 
```bash
./mvnw --no-transfer-progress -V -B -ff \ -Dsurefire.rerunFailingTestsCount=1 install
```